### DISCUSSION TITLE
Issue with package updates on Debian sid/unstable [SOLVED]

### DISCUSSION THREAD

**Discussion by edwardp3** [lxqt/lxqt#2374] [Q&A] [answered]
If anyone with the project uses Debian and is a developer, could they please look at this [Debian Forum thread](https://forums.debian.net/viewtopic.php?t=153858) (which I started)?

Debian's **task-lxqt-desktop** meta-package also installed (what I call) a minimal Plasma desktop (X11-only) along with numerous KDE-related packages. For the past week or so, there have been numerous packages unable to be updated (held back by apt), because other packages have not been updated.

So far, I have been unable to learn why the meta-package also installed Plasma and whether that was even necessary.

I would also like to know if there is a SAFE way to remove (at this point) both LXQt and Plasma, without breaking the system.

Thank you.

**Comment by tsujan** [comment, marked as answer]
Not being a Debian user, I can comment only on the relation between LXQt and KDE:

 1. LXQt doesn't depend on KDE. But KDE developers have created nice tools that are extensions of Qt. Those tools don't depend on KDE and are used by LXQt for not reinventing the wheel. KWindowSystem is an example.
 2. KWin can be used under LXQt — like any other modular WM. But KWin has some artificial KDE dependencies that should be installed alongside it.

**Reply by edwardp3** [reply]
@tsujan I ended up letting Synaptic do its thing. In the end, it actually removed all of the Plasma packages, including the KWin packages) and as of now, only two packages are being held back, one of which is a dependency for an LXQt-related package. The LXQt desktop continues to launch and work perfectly.

When I tried KWin_X11 on LXQt, I found it too slow.

**Reply by tsujan** [reply]
Good to know that you solved your problem.

> When I tried KWin_X11 on LXQt, I found it too slow.

Yes, KWin may be slow for some users. It depends on the system's graphics as well as KWin's settings (which can be changed by using KDE's systemsettings5).

**Reply by edwardp3** [reply]
Thank you.

On this particular desktop, it is using the on-board GPU, an AMD Radeon 3000 (RS780L), it has VGA and DVI outputs. I am using a DVI-to-HDMI cable to connect the monitor. The motherboard only came with two PCI Express x1 ports on it (both closed), so it's limited to an x1 external video card, should I decide to install one.. It's 12 years old, still works.  :)
